src/Volume.ts

```diff
@@ -286,21 +287,36 @@ export default class Volume {
     }
 
     if (shouldReload) {
-      this.loadNewData(onChannelLoaded);
+      await this.loadNewData(onChannelLoaded);
```

Comment:
My hope here was to consolidate data loading into the promise returned by `updateRequiredData`. Without the `await`, that promise resolves once loading has begun; with it, it resolves after loading is complete. Talking it through though, this is a relic from the brief moment yesterday when I thought that this promise would make it all the way out of the viewer API via the settings-setters that call it. Without that code, it doesn't make much sense in the first place.

On performance: a little research shows that `await`s do incur a smallish performance penalty, though (unless I'm really misunderstanding) that's all bookkeeping and runtime overhead rather than the waiting code actually blocking other stuff from executing. A spot like this shouldn't be too bad since it's a constant penalty, but I'll just remove this anyways